rudder-jetty/debian/preinst

```diff
@@ -0,0 +1,42 @@
+#!/bin/sh
+# preinst script for normation-openldap-framework
+#
+# see: dh_installdeb(1)
+
+set -e
+
+# summary of how this script can be called:
+#        * <new-preinst> `install'
+#        * <new-preinst> `install' <old-version>
+#        * <new-preinst> `upgrade' <old-version>
+#        * <old-preinst> `abort-upgrade' <new-version>
+# for details, see http://www.debian.org/doc/debian-policy/ or
+# the debian-policy package
+
+
+case "$1" in
+    install|upgrade)
+
+        # Prepare the migration of /etc/default/rudder-jetty
+        if [ $(grep -c '# WARNING #' /opt/rudder/etc/rudder-jetty.conf) -eq 0 ]
+        then
+            cp /opt/rudder/etc/rudder-jetty.conf /opt/rudder/etc/rudder-jetty.conf.migrate
```

Comment:
Because if the file in /opt/rudder/etc/rudder-jetty.conf does not begin with "######## WARNING #########", it means we still have the old file, and we need to keep the values in it to add them to the new one :)

This is why I do it in preinst